src/utils/theme/index.js

```diff
@@ -33,8 +36,16 @@ export const COLORS = {
   holytosca: '#00ADB5',
   lightBlue: '#2F80ED',
   gray1: '#C4C4C4',
-  gray100: '#F5F6F7',
-  gray500: '#69707C',
+  gray100: '#233343',
+  gray200: '#3A4F64',
+  gray300: '#52667A',
+  gray400: '#8C939F',
+  gray500: '#A5ACB8',
+  // gray100: '#FF00FF',
+  // gray200: '#FF00FF',
+  // gray300: '#FF00FF',
+  // gray400: '#FF00FF',
+  // gray500: '#FF00FF',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new colors `gray100` through `gray500` have been defined but commented out. If these are not being used, consider removing them to avoid cluttering the codebase.



<!-- This is an auto-generated comment by OSS CodeRabbit -->